main.cc

```diff
@@ -430,6 +441,8 @@ void render() {
     camera_params.ptr->aspect = (float)wnd.width / wnd.height;
     camera_params.bind(0, frame);
 
+    ddRenderIfaceGL->mvpMatrix = camera_params.ptr->view_proj_matrix;
```

Comment:
Don't read this back. It's quite possibly USWC, not cacheable.

Reply:
alright, thanks